coordinator/tests/routes/util.spec.ts

```diff
@@ -147,7 +147,7 @@ describe("handleResponse", () => {
     server.route([createRoute(spineResponse)])
 
     const response = await server.inject(
-      createRouteOptions(spineResponse, {"X-Untranslated-Response": "true"})
+      createRouteOptions(spineResponse, {"X-Raw-Response": "true"})
```

Comment:
Is the logic to support this in place yet?

Reply:
yep